### DISCUSSION TITLE
Devices dissapaered from my google home app

### DISCUSSION THREAD

**Discussion by JN-CLS** [andrei-tatar/node-red-contrib-smartnora#122] [General]
Hi,
After running about 4 to 5 months without issue, today most of my devices from my smartnora disappeared. I only have 5 left that are still working.  I know a discussion is ongoing on the a contribution fee but don't know if it had anything to do with this? (I will respond also to the inquiry on this)  Or maybe it is because I still was on the 1.0 instead the 1.5 version ?
Any info on the issue, and how to solve is very  welcome.  Thx

PS: 
Just saw the other discussion:
 
> So I integrated stripe for subscriptions. There are 2 prices available: 2 eur / month and 20 eur / year.
> If/When the number of subscriptions cover the business overhead, the price will get lower.
> I don't want to make a living out of NORA just covering the costs and a few beers :)
> After 1st December, users with no subscriptions will only get 5 devices, across all groups, synced with Google Home

Subscription for 20€ is reasonable and fine for me. Please guide me on how to subscribe.
Can I pay with Paypal ?

**Comment by andrei-tatar** [comment]
Hi, it is related to the contribution fee. As of last night, the free plan will limit the number of devices synchronized with Google Home to 5. If you want unlimited devices, you need to use a paid subscription (2 eur/month or 20 eur/year).

> Subscription for 20€ is reasonable and fine for me. Please guide me on how to subscribe.
> Can I pay with Paypal ?

You can visit smart-nora for more details: https://smart-nora.eu/my-nora
The payment integration is done with Stripe. It doesn't currently support paypal.

> You have no active subscription.
> This will limit the number of devices synchronized with Google Home to 5.

**Reply by JN-CLS** [reply]
Hi Andrei,
Subscription done. Will my devices get activated automatically ?

**Reply by andrei-tatar** [reply]
Yes, they should be.

**Comment by waschbaer77** [comment]
I had the same problem today. I've made a subscription now. I think 20€/year is okay.
BUT: Somebody could have mention this by E-Mail. So I don't have to waste time google around, restart services, etc. and searching for the problem.  I'm a paying customer 😉

**Reply by andrei-tatar** [reply]
In my defense, I did send a notification to everyone. I expected it to be less intrusive but I guess not everyone is subscribed to notifications.

**Reply by waschbaer77** [reply]
Okay, that's the problem. Anyway - Thanks for your great work. It helps me a lot.

**Comment by Heelderpeel** [comment]
same surprise for me today, after payment everything is fine again. thank you for this beautiful application.

**Comment by KenSpurling** [comment]
Same problem here but now fixed by taking a subscription. 
Pleased that Smart Nora has now moved to a paid for service for a reasonable charge. Hopefully it should now be able to operate for the foreseeable future. Best of luck

**Comment by Zephalon** [comment]
Well, theres two hours of my time I won't get back. 

20 bucks per year is totally fine but I had ZERO feedback what was wrong. My devices just appeared to be offline and I could not even change the device name any more.

**Reply by andrei-tatar** [reply]
Sorry for that. I did send a notification. I will send these kinds of updates by email next time.
